src/hashes.c

```diff
@@ -1320,17 +1416,23 @@ int hashes_init_stage1 (hashcat_ctx_t *hashcat_ctx)
         {
           hashes_cnt = hashes_parsed;
         }
-        else if (hashes_parsed == 0)
-        {
-          event_log_warning (hashcat_ctx, "No hashes loaded.");
-        }
-        else if (hashes_parsed == PARSER_HAVE_ERRNO)
-        {
-          event_log_warning (hashcat_ctx, "Hashfile '%s': %s", hashes->hashfile, strerror (errno));
-        }
         else
         {
-          event_log_warning (hashcat_ctx, "Hashfile '%s': %s", hashes->hashfile, strparser (hashes_parsed));
+          if (user_options->guess_hash_mode == false)
+          {
+            if (hashes_parsed == 0)
+            {
+	      event_log_warning (hashcat_ctx, "No hashes loaded.");
```

Comment:
Indentation is wrong.

```suggestion
              event_log_warning (hashcat_ctx, "No hashes loaded.");
```